bin/build_and_run_all_tests

```diff
@@ -122,8 +135,27 @@ fi
 
 
 
-# Configure, build & test
+# Configure, build & test for pdi
 
+cd "${TEST_DIR_WITH_PDI}"
 cmake -DDIST_PROFILE=Devel ${CMAKE_FLAGS} "${SRCDIR}"
 make ${MAKEFLAGS}
 ctest --output-on-failure --timeout 90 ${CTEST_FLAGS} ${EXCLUDED_PDI_TESTS:+-E $EXCLUDED_PDI_TESTS}
+
+RELATIVE_NO_PDI_PATH="no-pdi"
+if ldconfig -p | grep -q libparaconf; then
```

Comment:
paraconf might be installed but not system-wide. I'd prefer to trust the caller and use the provided info.

```suggestion
if [[ "x${PDI_LIBS}" = "xprovided" ]]; then
```